service-wfs-client/src/main/java/org/oskari/service/wfs/client/OskariFeatureClient.java

```diff
@@ -18,6 +18,7 @@
 public class OskariFeatureClient {
 
     protected static final String PROPERTY_NATIVE_SRS = "oskari.native.srs";
+    protected static final String PROPERTY_FORCE_GML = "forceGML";
```

Comment:
This same constant appears in OskariWFSClient

Reply:
removed